pkg/apis/virt/v1alpha1/types.go

```diff
@@ -114,6 +116,14 @@ type ContainerRootfsVolumeSource struct {
 	Size            resource.Quantity `json:"size"`
 }
 
+type PersistentVolumeClaimSource struct {
```

Comment:
This should be named to `PersistentVolumeClaimVolumeSource`?

Reply:
maybe not, if so  DataVolumeSource should be DataVolumeVolumeSource.